spec/integration/hawkular_client_spec.rb

```diff
@@ -267,6 +267,18 @@ module Hawkular::Client::RSpec
         end
       end
 
+      it 'Should work initializing with URI' do
+        uri = URI.parse HOST
+        opts = { tenant: 'hawkular' }
+        WebSocketVCR.configure do |c|
+          c.hook_uris = [HOST]
+        end
+        WebSocketVCR.record(example, self) do
+          the_client = Hawkular::Client.new(entrypoint: uri, credentials: @creds, options: opts)
+          expect { the_client.operations }.to_not raise_error
```

Comment:
The cassette is empty, because the `WebSocketVCR` records only the websocket communication, but in this case the WS connection wasn't opened. It's opened lazily during the first use of `the_client.operations` (https://git.io/vKMEh)

So either call the `the_client.operations` to open the WS connection, or remove the `WebSocketVCR.record(example, self) do`, this way it doesn't make much sense.